### PR TITLE
List links to various integrations on deployment docs landing page. A…

### PR DESCRIPTION
**Pull request by soulshake**
…dd tip on finding rack hostname via CLI or console.

## Release Playbook
- [ ] Rebase against master
- [ ] Code review
- [ ] Merge into master
- [ ] Verify changes at http://site-staging.convox.com
- [ ] Promote release on `site-production` in the `convox/production` Rack

### CODE REVIEW

**Comment by soulshake** [comment]
[Docs: Linkify integrations, add tips [easy]](https://trello.com/c/TmlsGMHO/1546-docs-linkify-integrations-add-tips-easy)

**Comment by soulshake** [comment]
[Docs: Linkify integrations, add tips [easy]](https://trello.com/c/TmlsGMHO/1546-docs-linkify-integrations-add-tips-easy)

**Comment by mattmanning** [comment]
@soulshake This looks like good information, but I don't understand why it's in the deployment doc. It seems out of place here.

**Comment by soulshake** [comment]
@mattmanning I thought the deployment doc made sense because I figured that's the step where people are most likely to need to know the Rack hostname. I agree that it does feel a little out of place, but I couldn't find a more logical location for it. Do you have a suggestion? Can also create a new page if needed.

**Comment by mattmanning** [comment]
We have a dedicated Integrations section in the docs, so maybe you could add an overview page there if you want to share info about integrations in general. You could possibly link to the DataDog doc in Monitoring an App.

Maybe we should have a doc like "Connecting to a Rack" or something like that under Management?  That doc could explain how to get the Rack hostname and api key. Those are needed both for adding a Rack to Console and for authenticating API requests send directly to the rack (not through Console).

**Comment by ddollar** [comment]
Let's probably drop the "Connecting to a Rack" page. Let's emphasize everything through Console and remove information about connecting directly to a Rack. The duality is confusing.

Power users can figure out the direct Rack connection thing. Maybe in the project README?

**Comment by soulshake** [comment]
@ddollar @mattmanning [warning: ideology ahead]

The main purpose of this `connecting to a Rack` page is to have a convenient place to link to in case someone, for whatever reason, needs to obtain their Rack hostname and/or API key. The most obvious, non-power-user use case is adding an existing Rack to Console. This isn't mutually exclusive with installing Racks through Console (e.g. it's needed if you want to access a Rack via more than one organization or account). 

I don't think it's an uncommon question (see slack logs), and we don't have a good page to link to when it's asked.

Regarding your comment that "Power users can figure out the direct Rack connection thing"--probably, but:

* having the `aws cloudformation` command right there in the docs is the kind of convenient touch that power users appreciate
* documentation is how regular users become power users. 

**TL;DR**

Telling people to go click around in AWS should be a last resort.

If you really don't want to have this very convenient `aws cloudformation` command in our docs, can I suggest we add the Rack hostname to the output of `convox rack`?

As always: If I haven't convinced you, I'm happy to defer.  I'll assume you still want this to be taken out unless I hear from you otherwise.

**Comment by ddollar** [comment]
I'm convinced. Leave it in.
